docs/src/main/sphinx/release/release-372.md

```diff
@@ -0,0 +1,140 @@
+# Release 372 (30 Feb 2022)
+
+## General
+
+* Add :func:`trim_array` function. ({issue}`11238`)
+* Improve handling of prepared statements with long query text by compressing
+  them within HTTP headers. This can be configured or disabled using the
+  ``protocol.v1.prepared-statement-compression.length-threshold`` configuration
+  property. ({issue}`11098`)
+* Improve performance of specific queries which compare table columns of type
+  `timestamp` with `date` literals. ({issue}`11170`)
+* Add redirection awareness for `ADD COLUMN`, `DROP TABLE`, `COMMENT` tasks.
+  ({issue} `11072`)
+* Remove support for reserved memory pool. Configuration property
+  `experimental.reserved-pool-disabled` can no longer be used. ({issue}`6677`)
+* Ensure memory is released completely after query completion. ({issue}`11030`)
+* Fix certain queries failing due to dictionary compacting error.
+  ({issue}`11080`)
+* Fix `SET SESSION` and `RESET SESSION` not working for catalogs which include
+  special characters in their name. ({issue}`11171`)
+
+## Security
+
+* Allow configuration of connetct and read timeouts for LDAP authentication.
+  ({issue}`10925`)
+
+## Docker image
+
+* Add a healthcheck to the Docker container image. ({issue}`10413`)
+
+## JDBC driver
+
+* Fix `DatabaseMetaData#getTables` and `DatabaseMetaData#getColumns` to include
+  views for Iceberg, Raptor, Accumulo and Blackhole connectors.
+  ({issue}`11063`, {issue}`11060`)
+
+## Base-JDBC connector library
+
+* Fix spurious query failures when metadata cache is not enabled, and data
```

Comment:
I don't think the comma between the two clauses is necessary. It makes the sentence sound weird.